lana/package.json

```diff
@@ -176,13 +176,15 @@
   },
   "dependencies": {
     "@apexdevtools/apex-ls": "^5.10.0",
+    "@apexdevtools/apex-parser": "^4.4.0",
     "@apexdevtools/sfdx-auth-helper": "^2.1.0",
     "@salesforce/apex-node": "^1.6.2"
   },
   "devDependencies": {
     "@types/node": "~22.16.3",
     "@types/vscode": "~1.99.1",
     "concurrently": "^9.2.0",
-    "typescript": "^5.9.2"
+    "typescript": "^5.9.2",
+    "antlr4ts": "0.5.0-alpha.4"
```

Comment:
I think this  is already a dep of `@apexdevtools/apex-parser` so not technically needed to be imported to use (version needs to be kept in sync this way).

If you want to explicitly define it, it should be a dep and not a dev dep.